src/legacy/ui/public/agg_types/__tests__/buckets/date_histogram/_params.js

```diff
@@ -87,56 +87,51 @@ describe('date_histogram params', function () {
       expect(output.params).to.have.property('fixed_interval', '30s');
     });
 
-    it('scales up the interval if it will make too many buckets', function () {
-      const timeBounds = getTimeBounds(30, 'm');
-      const output = writeInterval('s', timeBounds);
-      expect(output.params).to.have.property('fixed_interval', '10s');
-      expect(output.metricScaleText).to.be('second');
-      expect(output.metricScale).to.be(0.1);
-    });
+    describe('scaling behavior', () => {
 
-    it('does not scale down the interval', function () {
```

Comment:
We can still keep this test - it should test whether the interval is not scaled down even if `scaleMetricValues` is true if it's not necessary

Reply:
I might have been a bit to eager removing tests here :sweat_smile:  Added that test back.